internal/resources/grafana/resource_alerting_rule_group.go

```diff
@@ -235,7 +235,8 @@ func deleteAlertRuleGroup(ctx context.Context, data *schema.ResourceData, meta i
 	group := resp.Payload
 
 	for _, r := range group.Rules {
-		_, err := client.Provisioning.DeleteAlertRule(r.UID)
+		params := provisioning.DeleteAlertRuleParams{UID: r.UID}
```

Comment:
If you rebase, you can get rid of this change and the go mod update!

Reply:
Yeah! I saw that in your PR 😂